docs/reference/hardware_features.rst

```diff
@@ -0,0 +1,249 @@
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+      *
+        - :ref:`bfloat16 16-bit floating-point operations<rocm:precision_support_floating_point_types>`
+        - ❌
+        - ✅
+        - ❌
+        - ✅
+        - ✅
+        - ✅
+      *
+        - Support for :ref:`8-bit floating-point types <rocm:precision_support_floating_point_types>`
+        - ❌
+        - ❌
+        - ❌
+        - ❌
+        - ❌
+        - ✅
+      *
+        - Support for :ref:`tensor float32 <rocm:precision_support_floating_point_types>`
+        - ❌
+        - ❌
+        - ❌
+        - ❌
+        - ❌
+        - ✅
+      *
+        - Packed math with 16-bit floating point values
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+      *
+        - Packed math with 32-bit floating point values
+        - ❌
+        - ❌
+        - ❌
+        - ✅
+        - ❌
+        - ✅
+      *
+        - Matrix Cores
+        - ❌
+        - ✅
+        - ❌
+        - ✅
+        - ❌
+        - ✅
+      *
+        - On-Chip Error Correcting Code (ECC)
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+      *
+        - Maximum dimensionality of grid
+        - 3
+        - 3
+        - 3
+        - 3
+        - 3
+        - 3
+      *
+        - Maximum x-, y- or z-dimension of a grid
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+      *
+        - Maximum number of threads per grid
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+        - :math:`2^{32} - 1`
+      *
+        - Maximum x-, y- or z-dimension of a block
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+      *
+        - Maximum number of threads per block
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+        - :math:`1024`
+      *
+        - Wavefront size
+        - 32 [1]_
+        - 64
+        - 32 [1]_
+        - 64
+        - 32 [1]_
+        - 64
+      *
+        - Maximum number of resident blocks per compute unit
+        - 40 [1]_
+        - 32
+        - 32 [1]_
+        - 32
+        - 32 [1]_
+        - 32
+      *
+        - Maximum number of resident wavefronts per compute unit
+        - 40 [1]_
+        - 32
+        - 32 [1]_
+        - 32
+        - 32 [1]_
+        - 32
+      *
+        - Maximum number of resident threads per compute unit
+        - 1280 [2]_
+        - 2048
+        - 1024 [2]_
+        - 2048
+        - 1024 [2]_
+        - 2048
+      *
+        - Maximum number of 32-bit vector registers per thread
+        - RDNA1: 256
```

Comment:
Since the column head indicates the RDNA or CDNA architecture, do we need to repeat it here in the cell?

Reply:
Yep, overlooked that - that was for me to keep track of what column the values are for 😅